messages/se.json

```diff
@@ -14,7 +17,12 @@
     "development": "Utveckling",
     "project_management": "Projektledning",
     "url": "Link",
-    "collaborators": "Samarbeidspartnere"
+    "collaborators": "Samarbeidspartnere",
+    "customer_case_entry": {
+      "image": "bild",
+      "case": "Case",
+      "field": "Field"
```

Comment:
Needs a swedish translation of "field"

Reply:
I think we need to get someone speaking Swedish to look at the translations. I'll add something for now, but absolutely not sure if it's correct.